Pages/productlist.css

```diff
@@ -0,0 +1,124 @@
+.main-wrapper {
+  height: 90vh;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto;
```

Comment:
You can skip this line of code I guess it will anyway make rows implicitly. Correct me if I am wrong.

Reply:
Ya it will make rows implicitly, thanks for telling.